packages/artillery/test/cli/command-run.test.js

```diff
@@ -88,6 +88,49 @@ tap.test('Environment specified with -e should be used', async (t) => {
   t.ok(exitCode === 0 && !output.stdout.includes('ECONNREFUSED'));
 });
 
+tap.test('Can specify scenario to run by name', async (t) => {
+  const reportFile = 'report-with-scenario-by-name.json';
+  const reportFilePath = await getRootPath(reportFile);
+
+  const [exitCode, output] = await execute([
+    'run',
+    '--scenario-name',
+    'Test Scenario 2',
+    '-o',
+    `${reportFilePath}`,
+    'test/scripts/scenario-named/scenario.yml'
+  ]);
+
+  t.ok(
+    exitCode === 0 && output.stdout.includes('Successfully running scenario 2')
+  );
+  const json = JSON.parse(fs.readFileSync(reportFilePath, 'utf8'));
+
+  t.ok(
+    deleteFile(reportFilePath) &&
+      json.aggregate.counters['vusers.created_by_name.Test Scenario 2'] === 6
```

Comment:
This works to make sure that no other scenarios were run but would be nice to have a more explicit test for it.

```suggestion
      typeof json.aggregate.counters['vusers.created_by_name.Test Scenario 1'] === 'undefined'
```